src/metabase/driver/generic_sql/native.clj

```diff
@@ -27,9 +27,8 @@
              (.setAutoCommit jdbc-connection false)
              (try
                ;; Set the timezone if applicable
-               (when-let [timezone (driver/report-timezone)]
-                 (when (and (seq timezone)
-                            (contains? (driver/features driver) :set-timezone))
+               (when-let [timezone (:report-timezone settings)]
+                 (when (seq timezone)
```

Comment:
Not every database supports setting timezones, why would we want to stop taking that into account?